sql/solutions/lesson2.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: page
+title: Introduction to SQL Solutions
+---
+# Lesson 2 solutions to exercises
+
+## L2.1 Show the names of all the people. Sort the result by name in ascending (alphabetical) order, in case of equal name sort by country of residence in alphabetical order.
+
+```SQL
+SELECT name
+  FROM people
+ ORDER BY name ASC, country_of_residence ASC;
+```
+
+##  L2.2 Show the name and the country of birth of the person with highest number of children.
+
+```SQL
+SELECT name, country_of_birth
+  FROM people
+ ORDER BY number_of_children DESC
+ LIMIT 1;
+```
+
+##  L2.3 Show the country of birth of the oldest person.
+
+```SQL
+SELECT country_of_birth
+  FROM people
+ ORDER BY age DESC
+ LIMIT 1;
+```
+
+##  L2.4 Show the age of the oldest person.
+
+```SQL
+SELECT age
+  FROM people
+ ORDER BY age DESC
+ LIMIT 1;
+```
+
+##  L2.5 Show the location of cities in UK.
```

Comment:
I think there is an extra **of** here. it should read: Show the **locations cities** in UK.

right?